contrib/test_jenkins.sh

```diff
@@ -979,6 +1024,86 @@ run_ucx_perftest() {
 
 		unset CUDA_VISIBLE_DEVICES
 	fi
+
+	# run rocm tests if rocm module was loaded and GPU is found
```

Comment:
can we introduce a function that accepts all needed arguments (e.g. kernel module ("amdgpu", "nv_peer_mem"), env variable name ("HIP_VISIBLE_DEVICES", "CUDA_VISIBLE_DEVICES") and etc) and run CUDA/ROCm perftests?

Reply:
Can be a bit tricky due the number of different environment variables. Wanted to fix the errors first before trying to refactor.

Reply:
agree with @dmitrygx , this is too much ~code~script duplication..